test-butler-demo/src/main/res/layout/activity_main.xml

```diff
@@ -14,12 +14,18 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<LinearLayout
+<RelativeLayout
     android:id="@+id/activity_main"
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
     tools:context="com.linkedin.android.testbutler.demo.MainActivity">
-</LinearLayout>
+
+    <EditText
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/app_name"
+        android:layout_alignParentBottom="true"
```

Comment:
It doesn't _really_ matter for this, but I think you could keep the `LinearLayout` and use a gravity=bottom to achieve the same effect